dix/cm/cm.c

```diff
@@ -600,6 +605,16 @@ M0_INTERNAL int m0_dix_cm_data_next(struct m0_cm *cm, struct m0_cm_cp *cp)
 			dix_cp->dc_ctg_op_flags |= COF_CREATE;
 			dix_cp->dc_is_local      = true;
 			dcm->dcm_cp_in_progress  = true;
+
+			/* XXX The following is used to print key & val */
+			/*
```

Comment:
Question:
Is this supposed to be commented?

Reply:
This is helpful for debugging some defects.

But in normal case, if the key and value is very long, this will create some performance impact. Also this message is not very useful if we are not going to debug specific bug.
So, I added it but disabled it.

Reply:
Anyone, if want to debug related issues, e.g. to see all the keys and values (Not so many) in some index, can enable this.